tests/test_main.py

```diff
@@ -574,3 +577,20 @@ def test_create_finding_data_regular_project_from_cli_project_with_custom_name()
     assert tf_finding['mappings'][1]['mappingType'] == 'CWE'
     assert tf_finding['mappings'][1]['value'] == '502'
     assert tf_finding['mappings'][1]['primary'] is True
+
+
+def test_module_version_matches_pyproject_version():
+    """So we don't update the version in one place only and not both"""
+    version_from_package_init = __version__
+
+    # this is so that the test finds the pyproject.toml file when run from the command line or from within Pycharm
+    this_directory = os.path.dirname(os.path.realpath(__file__))
+    pyproject_toml_path = this_directory + '/../pyproject.toml'
```

Comment:
This assumes you're on a system where forward slashes are the path separator. Best to use `os.path.join` to make things like this cross platform.

Reply:
fixed